api/src/main/java/org/openmrs/module/registrationcore/api/impl/PatientActionListener.java

```diff
@@ -56,9 +87,7 @@ public void run() {
 
     protected Patient extractPatient(Message message) {
         validateMessage(message);
-
-        String patientUuid = getMessagePropertyValue(message, RegistrationCoreConstants.KEY_PATIENT_UUID);
-
+        String patientUuid = getMessagePropertyValue(message, "uuid"); // Property name referenced from org.openmrs.event.EventEngine.fireEvent(javax.jms.Destination, java.lang.Object)
```

Comment:
If you still need this comment, it would be better to put it at the top of the line.

Reply:
shifted